wagtail/users/tests/test_admin_views.py

```diff
@@ -1328,6 +1328,9 @@ def test_user_cannot_escalate_privileges(self):
 
 class TestGroupIndexView(AdminTemplateTestUtils, WagtailTestUtils, TestCase):
     def setUp(self):
+        # This group should display after the default groups but will display
+        # before them if default_ordering is lost.
+        Group.objects.create(name="Photographers")
```

Comment:
I think we can move this to the `test_default_ordering` test directly, as it's not needed for the others